packages/field-base/src/styles/sr-only-styles.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright (c) 2021 - 2023 Vaadin Ltd..
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { css } from 'lit';
+
+export const screenReaderOnly = css`
+  .sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    border: 0;
+  }
```

Comment:
This is a complex topic, wanna e.g. use a and link to a common used one like https://gist.github.com/ffoodd/000b59f431e3e64e4ce1a24d5bb36034

Reply:
Changed the styles as suggested.